setup.cfg

```diff
@@ -34,7 +34,18 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .tox, .git, __pycache__, .ipynb_checkpoints
-ignore = D105, D107, SFS2  # SFS2 can be changed to SFS3 if supporting >=3.6 only
+extend-ignore =
```

Comment:
@csala mentioned that we should comment what each lint error means. You could just copy paste from my PR's (I think we are ignoring the same errors).